pkg/sql/testdata/pg_catalog_tables_from_postgres.json

```diff
@@ -3657,6 +3664,178 @@
         "expectedDataType": null
       }
     },
+    "pg_stat_database": {
+      "blk_read_time": {
+        "oid": 701,
+        "dataType": "float8",
+        "expectedOid": null,
```

Comment:
related to above: in the future, we should see if we can avoid putting expectedOid and expectedDataType in this file. since this is not a diff file, these will always be null, so it would be better to not have them here at all

Reply:
Same reason, they share the same data structures, as these files were not initially made for human read.